CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing
+
+We're always looking to improve this project, open source contribution is encouraged so long as they adhere to these
+guidelines.
+
+## Pull Requests
+
+The Flagsmith team will be monitoring for pull requests. When we get one, a member of team will test the work against
+our internal uses and sign off on the changes. From here, we'll either merge the pull request or provide feedback
+suggesting the next steps.
+
+### A couple things to keep in mind
+
+- If you've changed APIs, update the documentation.
+- Keep the code style (indents, wrapping) consistent.
+- If your PR involves a lot of commits, squash them using `git rebase -i` as this makes it easier for us to review.
+- Keep lines under 80 characters.
```

Comment:
nit: I feel that

> - Keep the code style (indents, wrapping) consistent.

and

> - Keep lines under 80 characters.

are outdated as we use linters in CI for most of our projects.